### PR TITLE
Fixed the sort icons arrow pointing the wrong direction TINY-5097

### PR DESCRIPTION
**Pull request by lostkeys**
Related Ticket: [TINY-5097](https://ephocks.atlassian.net/browse/TINY-5097)

Description of Changes:
Fixed the sort icons arrow pointing the wrong direction

Pre-checks:
* [x] Changelog entry added
* [x] ~~Tests have been added (if applicable)~~
* [x] ~~Branch prefixed with `feature/` for new features (if applicable)~~
* [x] ~~License headers added on new files (if applicable)~~

Review:
* [x] Milestone set
* [ ] Review comments resolved

GitHub issues (if applicable): n/a

### CODE REVIEW

File: modules/oxide-icons-default/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tinymce/oxide-icons-default",
-  "version": "1.3.3",
+  "version": "1.3.4",
```
Comment (lnewson) [inline modules/oxide-icons-default/package.json:3]: You should never manually bump versions in the monorepo, as that needs to be done by `lerna` which we do at release time (see the contributing docs for more details). Sorry I know this is inconsistent across the various things.

If it helps, here are the general rules to follow:
- Is it a patch version bump? then don't manually bump the version
- Is it a minor/major version bump? then manually bump the version, except in monorepos

- [ ] Revert manual package.json bump

**Comment by lostkeys** [comment]
This illustrates the issues with icons. They doesn't add much value here, rather they add confusion. What about just updating the text strings to "Sort columns by A→Z" or similar

**Comment by lostkeys** [comment]
Personally, I never figured out sort arrows and what accends or decends. I click one of them, if it's the wrong one, I click the other one.

I prefer the Google Sheet approach. just say A→Z with no icon
image

Adding excel's icons as well for reference
image

**Comment by metricjs** [comment]
> Personally, I never figured out sort arrows and what accends or decends. I click one of them, if it's the wrong one, I click the other one.

Amusingly, when we discussed this as a team, someone said the same thing.

**Comment by lnewson** [comment]
Worth noting that we can't use `A -> Z` in the text, as we allow sorting based on varying algorithms (eg alphanumeric, numeric, etc...).

**Comment by lostkeys** [comment]
@lnewson I'd say A-Z is more figuratively than literally. I think it would be fine

**Comment by lostkeys** [comment]
I'm closing this PR. If we're doing sort icons, it will be done in a new PR/Branch.

**Comment by lnewson** [comment]
We will also need icons for any toolbar buttons, so the text based label isn't a viable solution.

**Comment by lnewson** [comment]
We should also consider removing the current icons given the issues and ongoing  discussion.

**Comment by lostkeys** [comment]
Customers can easily add their own icons. Yes, we'll need to remove the existig ones, good reminder.

**Pull request closed without merging**